docs/5.0/admin-guide.md

```diff
@@ -356,10 +368,8 @@ start using U2F:
 
 * Enable U2F in Teleport configuration `/etc/teleport.yaml` .
 
-* For CLI-based logins you have to install [u2f-host](https://developers.yubico.com/libu2f-host/) utility.
-
-* For web-based logins you have to use Google Chrome and Firefox 67 or greater, are the only
-   supported U2F browsers at this time.
```

Comment:
Do these comments apply to Teleport 5.0 or Teleport 6? It might be time for a new version as we get ready for a release.

Reply:
They're being deleted from the docs so shouldn't be very relevant?

Reply:
It would be nice to have a separate `docs/latest` folder for doc updates for "next release".

Right now, `docs/5.0` is basically that - it's no longer the actual 5.0 docs.
The actual 5.0 docs are in `branch/5.0`: what was cut during the release + any backports since then.

To answer your first question: removing `u2f-host` docs is new in 6, but browser compatibility message is applicable to any release.